transforms/language/doc_chunk/python/src/doc_chunk_transform.py

```diff
@@ -48,9 +52,11 @@ def __str__(self):
 
 
 default_content_column_name = "contents"
+default_doc_id_column_name = "doc_id"
```

Comment:
Is this the right value?  I think ededup uses `document_id`.  Should we try and standardize?

Reply:
very good catch! let's align it here.